backend/src/main/resources/application.properties

```diff
@@ -4,7 +4,7 @@ spring.datasource.password=fancyrats
 
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQLDialect
 spring.jpa.hibernate.ddl-auto=update
-spring.sql.init.mode=always
+#spring.sql.init.mode=always
```

Comment:
I commented this line out because the data.sql line was running before the entity tables were getting built so after it bootruns the first time and the database has the badge table you can comment this line back in and you will have the badges